playbooks/common/openshift-grafana/registry.yml

```diff
@@ -0,0 +1,9 @@
+---
+- include: config.yml
+
+- name: Initialize GlusterFS registry PV and PVC vars
+  hosts: oo_first_master
+  tags: hosted
+  tasks:
+  - set_fact:
+      gf_ds_name: {{ grafana_datasource_name }}
```

Comment:
There is no documentation with this PR. What is this `registry.yml` file for?

Reply:
initially I just setup the openshift-grafana dir based on other example.
I probably missed some cleanups

Reply:
@tbielawa I see lots of commons with that stack of files, Is it not required ?